WalletWasabi.Gui/Behaviors/CommandOnDoubleClickBehavior.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Avalonia.Controls;
+using System.Reactive.Disposables;
+
+namespace WalletWasabi.Gui.Behaviors
+{
+    public class CommandOnDoubleClickBehavior : CommandBasedBehavior<Control>
+    {
+        private CompositeDisposable _disposables;
```

Comment:
Please use the disposable pattern we use at other `OnAttached()` `OnDetaching()` classes.